.github/workflows/report.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         uses: exasol/python-toolbox/.github/actions/python-environment@0.14.0
```

Comment:
suggestion: could update these to use newer versions, but not strictly necessary
```suggestion
        uses: exasol/python-toolbox/.github/actions/python-environment@0.20.0
```

Reply:
Yes, why not?